sdk/invoke.js

```diff
@@ -30,13 +30,15 @@ const invoke = async function invoke(funcName, apiKey, funcData) {
     let parsedError;
     try {
       parsedError = JSON.parse(err.error);
+      const errorOrMsg = parsedError.error || parsedError.message;
+      if (parsedError.stack && errorOrMsg) {
+        throw new Error(`${errorOrMsg}\n${parsedError.stack}`);
```

Comment:
this throw will be caught in line 40, rendering the error useless.
You probably want to rewrite this block with try parse and if parsedError instead to get to a common error at the borrom